src/main/java/impl/ReaderFromFileImpl.java

```diff
@@ -0,0 +1,18 @@
+package impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import service.ReaderFromFile;
+
+public class ReaderFromFileImpl implements ReaderFromFile {
+    @Override
+    public List<String> read(String fromFileName) {
+        try {
+            return Files.readAllLines(Path.of(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file " + fromFileName, e);
+        }
```

Comment:
It's good to see that you're using try-with-resources and throwing a RuntimeException with an informative message in case of an IOException. This adheres to the checklist recommendations.